index.html

```diff
@@ -4530,9 +4542,9 @@ <h2>
           </li>
           <li>If the <a data-lt=
           "PaymentOptions.requestShipping">requestShipping</a> value of
-          |request|.<a>[[\options]]</a> is true, then if the <a data-lt=
-          "PaymentRequest.shippingAddress">shippingAddress</a> attribute of
-          |request| is null or if the <a data-lt=
+          |request|.[=PaymentRequest/[[options]]=] is true, then if the
+          <a data-lt="PaymentRequest.shippingAddress">shippingAddress</a>
```

Comment:
```suggestion
          {{PaymentRequest/shippingAddress}}
```

Could you check if there are others like this one? We should kill any data-lt="Foo.Bar" ones.

Reply:
same as https://github.com/w3c/payment-request/pull/894#discussion_r370455253 ^^